codes/dart/chapter_computational_complexity/time_complexity.dart

```diff
@@ -0,0 +1,166 @@
+/**
+ * File: time_complexity
+ * Created Time: 2023-02-12
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+
+class TimeComplexity {
```

Comment:
Remove class if it is uenecessary.

(In Java, we have to create a class but it is not elegant.)